index.html

```diff
@@ -251,33 +248,33 @@
 
                 return `
                     <h1>Your data</h1>
-                    <p>
+
                     <div><strong>Birth year</strong>: ${b_year} </div>
                     <div><strong>Birth month</strong>: ${b_month} </div>
                     <div><strong>Native language</strong>: ${n_lang} </div>
                     <div><strong>Multilingual</strong>: ${bilingual} </div>
                     <div><strong>Dyslexic</strong>: ${dyslexic} </div>
                     <div><strong>Gender</strong>: ${gender} </div>
                     <div><strong>Hand preference</strong>: ${hand_pref} </div>
-                    </p>
+
                     <BR><BR>
                     <p>Is this information correct?</p>
-                    ` 
+                    `;
             },
             choices: YES_OR_NO_BUTTON_TEXT,
             response_ends_trial: true,
             on_finish: function(data){
                 // Repeat the survey if yes (0) was not pressed
-                repeat_survey = data.button_pressed != 0
+                repeat_survey = data.button_pressed != 0;
             }
         };
 
         let select_keyboard_layout = {
             type: 'html-button-response',
-            stimulus: '<h2>Choose the keyboardlayout that looks most like yours</h2>',
+            stimulus: '<h2>Choose the keyboard layout that looks most like yours</h2>',
             choices: KEYBOARD_CHOICES,
             prompt: `
-                Focus on the first 6 <b>letters</b> in the top left of your 
+                Focus on the first 6 <strong>letters</strong> in the top left of your
```

Comment:
<b> is in principe prima, maar je gebruikt elders al <strong>, wat beter is. Dus vandaar ;)

Reply:
yes!